projects/nodejs.cmake

```diff
@@ -72,6 +72,8 @@ function(build_nodejs_ver ver)
     elseif(${BUILD_PLATFORM} STREQUAL "32")
       set(destcpu arm)
     endif()
+  elseif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
```

Comment:
so, aarch64 is the same thing as arm 64. I realize that at https://github.com/smanders/externpro/pull/299/files#diff-086be0a3150147469aacc49f836ce75704c3f20a6519fda22e73550b1f8535ccR71 we check for arm and 64, but its my understanding that while one arm64 processor will give aarch64, and others will give arm with the build platform as 64. So I just left the other check in